### ISSUE TITLE
Users reported Carthage install is broken on the latest version

### ISSUE THREAD

**Issue by bruno-garcia** [getsentry/sentry-cocoa#1060]
An update from 6.0.6 to 6.2.1 fails with:

> A shell task (/usr/bin/xcrun codesign --force --sign - --preserve-metadata=identifier,entitlements /Users/abc/Library/Developer/Xcode/DerivedData/App-ffvyradluhvbddeywqxepqcfsjyt/Build/Products/Release/App.app/Contents/Frameworks/Sentry.framework) failed with exit code 1:
/Users/abc/Library/Developer/Xcode/DerivedData/App-ffvyradluhvbddeywqxepqcfsjyt/Build/Products/Release/App.app/Contents/Frameworks/Sentry.framework: bundle format is ambiguous (could be app or framework)

Related to #1053

**Comment by philipphofmann** [comment]
I'm having trouble reproducing this. I set up a sample project with Carthage and 6.0.6, 6.2.1, and 7.0.0-beta.0 work fine. 

Maybe this could be related to [migrating a project from framework bundles to XCFrameworks](https://github.com/Carthage/Carthage#migrating-a-project-from-framework-bundles-to-xcframeworks). I opened up a PR to improve the docs on Carthage with Sentry, see https://github.com/getsentry/sentry-docs/pull/3452. Maybe this helps users. If it's related to Carthage there is nothing we can do. Our users must solve this on their side.
